### PR TITLE
Open selection in new [background] tab with shift+o

### PR DESCRIPTION
**Pull request by vanniktech**
That's the first part in regards to #1240

### CODE REVIEW

File: source/features/add-selection-in-new-tab.js [hunk 1]
```diff
@@ -7,7 +7,10 @@ export default function () {
 	document.addEventListener('keypress', event => {
 		const selected = select('.navigation-focus .js-navigation-open[href]');
 		if (selected && event.key === 'O') {
-			window.open(selected, '_blank');
+			browser.runtime.sendMessage({
+				urls: [ selected.href ],
+				action: 'openAllInTabs'
```
Comment (vanniktech) [inline source/features/add-selection-in-new-tab.js]: Let me know whether this is fine or not.

File: source/features/add-selection-in-new-tab.js [hunk 2]
```diff
@@ -7,7 +7,10 @@ export default function () {
 	document.addEventListener('keypress', event => {
 		const selected = select('.navigation-focus .js-navigation-open[href]');
 		if (selected && event.key === 'O') {
-			window.open(selected, '_blank');
+			browser.runtime.sendMessage({
+				urls: [selected.href],
+				action: 'openAllInTabs'
```
Comment (vanniktech) [inline source/features/add-selection-in-new-tab.js]: Let me know whether we can reuse the action name or not.

**Comment by vanniktech** [comment]
I tested it locally out and it worked.

**Comment by bfred-it** [comment]
Thanks @vanniktech! Glad to see you in the PR section :D

**Pull request merged**